lib/dms.js

```diff
@@ -46,9 +46,47 @@ class DmsModel {
     await Promise.all(datapackage.resources.map(async resource => {
       resource.views = await this.getResourceViews(resource.id)
     }))
+
+    // Fetch dictionary fields for the resources
+    await Promise.all(datapackage.resources.map(async resource => {
+      //TODO Check resource type and make
```

Comment:
Yes, we don't need to call this API if a resource isn't `datastore_active` - simple if statement would fix this.